design/Proposal for View Schema.md

```diff
@@ -0,0 +1,57 @@
+﻿
+
+
+
+## Schema definition for a View
+This is proposed  specification for a  schema definition of a view.  This is based on what a Dremio view schema would need to save along with the SQL text string describing the view. The goal is this schema could be used generally as well. 
+Described is an outline of the classes for the types and attributes for the view fields. 
+
+(This schema is a data member of the DremioView definition )
+
+A ViewSchema is a list of ViewFields and maintains a versionId to handle future version changes if any.
+
+    public final class ViewSchema implements Externalizable, Message<ViewSchema>, Schema<ViewSchema>  {      
+        private Integer versionId;  
```

Comment:
why a version for the schema? isn't that handled by Nessie git-like structure already?